src/main/java/io/kabanero/api/InstanceEndpoints.java

```diff
@@ -79,4 +88,36 @@ public Response listStacks(@PathParam("instanceName") String instanceName) throw
         }
         return Response.ok(stacks).build();
     }
-}
+
+    @GET
+    @Path("{instanceName}/admin")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response isAdmin(@PathParam("instanceName") String instanceName) throws IOException, ApiException, GeneralSecurityException {
+        UserProfile userProfile = UserProfileManager.getUserProfile();
+        String token = userProfile.getAccessToken();
+        GitHubClient client = new GitHubClient();
+        client.setOAuth2Token(token);
+
+        Kabanero instance = KabaneroClient.getAnInstance(instanceName);
+        String instanceGithubOrg = instance.getSpec().getGithub().getOrganization();
+        List<String> instanceGithubTeams = instance.getSpec().getGithub().getTeams();
+        Boolean isAdmin = false;
+
+        TeamService teamService = new TeamService(client);
+        List<Team> teams = teamService.getTeams(instanceGithubOrg);
+
+        for (Team team : teams) {
+            for (String teamName : instanceGithubTeams) {
+                if (teamName.equals(team.getName())) {
+                    isAdmin = teamService.isMember(team.getId(), new UserService(client).getUser().getLogin());
+                }
+            }
+        }
```

Comment:
might be helpful to name variables a bit more descriptive here - it was hard to follow along when `team.getName()` and `teamName` are two different things. Maybe `orgTeam` and `kabaneroAdminTeam`?